From 378b2c5678a753d68f9a8cdfb54c6a79a634bda9 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 7 Aug 2020 13:15:36 +0800
Subject: [PATCH] Test case for Driver - Dsdt.asl file

Description: Change the 'WORDBusNumber' defined in the TestPkg/AcpiTables/Dsdt.asl

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/AcpiTables/Dsdt.asl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/TestPkg/AcpiTables/Dsdt.asl b/TestPkg/AcpiTables/Dsdt.asl
index 383d72af54..dc76128afe 100644
--- a/TestPkg/AcpiTables/Dsdt.asl
+++ b/TestPkg/AcpiTables/Dsdt.asl
@@ -40,9 +40,9 @@ DefinitionBlock ("Dsdt.aml", "DSDT", 1, "INTEL ", "OVMF    ", 4) {
           PosDecode,             // PosDecode
           0x0000,                // Granularity
           0x0000,                // Min
-          0x00FF,                // Max
+          0x00FE,                // Max
           0x0000,                // Translation
-          0x0100                 // Range Length = Max-Min+1
+          0x00FF                 // Range Length = Max-Min+1
           )
 
         IO (Decode16, 0xCF8, 0xCF8, 0x01, 0x08)       //Consumed resource (0xCF8-0xCFF)
-- 
2.27.0.windows.1

